DESIGN.md

```diff
@@ -0,0 +1,107 @@
+# ODD
+
+## Introduction
+
+The ODD SDK is a local-first toolkit that helps developers build systems that store data in a secure, reliable, and decentralised way. The focus is on the user, providing them with sufficient authority while still being convenient.
+
+ODD stands for Open Distributed Data. We want our data accessible from anywhere and distributed, meaning that it can live partially or fully in multiple places. Ideally as much as possible of this data lives on the devices of the owner of the data and communication of this data would be done in a decentralised peer-to-peer fashion.
+
+We also prefer not to rely on centralised services when providing users and devices with the needed authority. When a user starts using a new device, their existing device should be authorise the new one without any intervention from a central service. This can be done by giving the new device one or more tickets that can be verified on the spot.
```

Comment:
```suggestion
We also prefer not to rely on centralised services when providing users and devices with the needed authority. When a user starts using a new device, their existing device should authorise the new one without any intervention from a central service. This can be done by giving the new device one or more tickets that can be verified on the spot.
```